index.js

```diff
@@ -222,6 +222,13 @@ class Entity {
       .getEntities(params)
       .then(response => _.map(response["entities"], entity => Entity.build(entity)));
   }
+
+  static getCollectionsByEntityId(client, entityId, params) {
+    return client
+      .getCollectionsByEntityId(entityId, params)
+      .then(response => _.map(response["collections"], collection => Collection.build(collection)));
```

Comment:
you don't need _.map().

Just use response["collections"].map()

Reply:
done the change for the above one.
however, all the other api calls are still using _.map(), do you want me to refactor all of them as well?

Reply:
If you can 😄 that be great